pkg/registry/apis/alerting/notifications/timeinterval/conversions.go

```diff
@@ -0,0 +1,83 @@
+package timeinterval
+
+import (
+	"encoding/json"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	model "github.com/grafana/grafana/pkg/apis/alerting_notifications/v0alpha1"
+	"github.com/grafana/grafana/pkg/services/apiserver/endpoints/request"
+	"github.com/grafana/grafana/pkg/services/ngalert/api/tooling/definitions"
+)
+
+func convertToK8sResources(orgID int64, intervals []definitions.MuteTimeInterval, namespacer request.NamespaceMapper) (*model.TimeIntervalList, error) {
+	data, err := json.Marshal(intervals)
+	if err != nil {
+		return nil, err
+	}
+	var specs []model.TimeIntervalSpec
+	err = json.Unmarshal(data, &specs)
+	if err != nil {
+		return nil, err
+	}
+	result := &model.TimeIntervalList{}
```

Comment:
The usage of resource models is very basic here. In the future we will add version, timestamps etc.